docs/contributing.rst

```diff
@@ -147,34 +153,68 @@ Some tips for reviewing
 
 
 .. code-block:: python
-
     def znl_mean(ar):
         return ar.mean('xt_ocean')
-
-has a few issues. First, the names don't read English. The method does not have any documentation nor is self-explanatory. Further, the method assumes that ``xt_ocean`` is a coordinate of the data array.
+has a few issues. First, the names aren't easily understood. The method does not have any documentation, nor is it self-explanatory. Furthermore, the method assumes that ``xt_ocean`` is a coordinate of the data array; hard coding dimension names is fragile to future changes.
 
 A much better version, free from all the cons mentioned above, is:
 
 .. code-block:: python
-
     def zonal_mean(dataarray):
         '''
         Returns the (numerical) zonal mean of `dataarray`, i.e., its mean along latitude circles.
-
             Parameters:
                     dataarray (xarray.dataarray): An xarray dataarray
-
             Returns:
                     xarray.dataarray: The (numerical) zonal mean of `dataarray`
         '''
-
         return dataarray.cf.mean('longitude')
-     
-
 * Ensure that the notebook runs! To do that:
 
   - Clone the repository or the fork that the PR was made from;
   - Checkout the appropriate branch;
   - Ensure that the notebook runs when a **new** kernel is launched. Ensure that all cells run in sequential order, and that all cell outputs are evaluated.
 
 .. _existing Pull Requests: https://github.com/COSIMA/cosima-recipes/pulls
```

Comment:
It's not clear to me what the point of this is. As far as I can tell, it doesn't render in the html version of the docs. Perhaps @angus-g or @navidcy know why it is here?

Reply:
This is defining a link target that is used a couple of times in the preceding text. rST is peculiar!

Reply:
Why did you remove empty lines though?

Reply:
> Why did you remove empty lines though?

By accident :(

Reply:
Blank lines reinstated (apart from the one before the `return` in `zonal_mean(dataarray)`, which seems superfluous. Happy to discuss though.

Reply:
okie!

Reply:
(I think some of the other empty lines if not there the rst wasn't rendering properly....)